contracts/ROSCA.sol

```diff
@@ -600,15 +606,18 @@ contract ROSCA {
    * this sends all the funds to the foreperson by selfdestructing this contract.
    */
   function emergencyWithdrawal() onlyFromForeperson onlyIfEscapeHatchActive external {
-    LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+	  LogEmergencyWithdrawalPerformed(getBalance(), currentRound);
+    bool failedSendingFunds = false;
     // Send everything, including potential fees, to foreperson to disperse offline to participants.
     bool isEthRosca = (tokenContract == address(0));
     if (!isEthRosca) {
```

Comment:
if isEthRosca is true?

Reply:
this is correct, we only want to call `transfer` method of the token contract, if the rosca is a token rosca